### DISCUSSION TITLE
[2022/02/01] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#190] [Archive]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/189 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by NotFounds** [comment]
## コードには型アノテーションよりも要件アノテーションを増やせ！/harajukuts2 - Speaker Deck 
https://speakerdeck.com/okunokentaro/harajukuts2

**Comment by NotFounds** [comment]
## （ネタがなければ）
https://developers.raycast.com/

**Comment by kobayang** [comment]
フロントエンドのモノレポ構成はスケーリングの夢を見るか
https://cybozu.github.io/frontend-expert/posts/considerations-for-monorepo

**Comment by kobayang** [comment]
状態、結合、複雑性、コード量の順に最適化する
https://ohbarye.hatenablog.jp/entry/2022/01/31/state-coupling-complexity-code

**Comment by chloe463** [comment]
## 便利ツールの話

- [DevToys の Mac 版](https://twitter.com/yuki_obuchi/status/1488159986450837505?s=20&t=SXKaoBHMKFvZM_mWo9yVhw)
- [RayCast](https://www.raycast.com/)
	- デフォルト設定でも結構使える
	- Google カレンダー連携がすごい
- [Better touch tool](https://folivora.ai/)
	- 2本指を置いた状態で片方の指タップとかをアクションに割り当てたり柔軟

**Comment by izumin5210** [comment]
> webpack build のメトリクスを収集したり、それを datadog に送ったりできる plugin らしい
> ビルド時間はもちろん、chunk size や「どの loadaer に時間かかってるか」「module が持つ依存の数」なんかもとれるっぽい
> https://github.com/DataDog/build-plugin

https://wantedly.slack.com/archives/CA46K091P/p1643690529323609
